### DISCUSSION TITLE
docs/3.x/api-reference/miscellaneous/objects/domain

### DISCUSSION THREAD

**Discussion by giscus** [saleor/saleor-docs#761] [General]
# docs/3.x/api-reference/miscellaneous/objects/domain

Represents shop's domain.

https://docs.saleor.io/docs/3.x/api-reference/miscellaneous/objects/domain

**Comment by simon-online** [comment]
"Represents shop's domain." is misleading, it's actually the Saleor API domain name e.g. `api.example.com` but "shop's domain" makes people think it is the main shop domain eg. `example.com`

**Reply by patrys** [reply]
Good catch! It is indeed how Saleor constructs URLs to its API endpoints.
